### DISCUSSION TITLE
P4952 [USACO04MAR] Financial Aid 题解 — scp020 的博客

### DISCUSSION THREAD

**Discussion by giscus** [scp020/scp020.github.io#1] [General]
# P4952 [USACO04MAR] Financial Aid 题解 — scp020 的博客

模拟赛考到的题，放到第二题的位置我不是很理解。

https://scp020.github.io/2023/10/04/solution-p4952/

**Comment by scp020** [comment]
test测试

~来呀~

**Comment by StevehimYu** [comment]
$test$
$$114514 ^1919810\dfrac{1}{0}$$
> SCP020

- qqq
- 222

1. awa
2. qwq

**Reply by StevehimYu** [reply]
好好好github难得快一回

**Reply by scp020** [reply]
因为我的giscus是挂在镜像网站上的。

**Reply by StevehimYu** [reply]
666

**Reply by StevehimYu** [reply]
$\text{SCP020 AKA IOI !}$

**Reply by scp020** [reply]
@StevehimYu 6

**Comment by scp020** [comment]
@StevehimYu AKA（

**Reply by StevehimYu** [reply]
:zap:

**Reply by StevehimYu** [reply]
事实上这是一个正反都能读的串，AKA没意思/doge

SCP020 AK IOI
IOI AK SCP020

**Reply by StevehimYu** [reply]
@scp020

**Reply by scp020** [reply]
@StevehimYu 我谢谢你
